docs/content-cards.rst

```diff
@@ -0,0 +1,69 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.. _content-cards:
+
+=================================
+Using External Content Cards Data
+=================================
+
+The `www-admin repo`_ contains data files and images that are synced to
+bedrock and available for use on any page. The docs for updating said
+data is available via that repo, but this page will explain how to use
+the cards data once it's in the bedrock database.
+
+Add to a View
+-------------
+
+The easiest way to make the data available to a page is to add the
+``page_content_cards`` variable to the template context:
+
+.. code-block:: python
+
+    from bedrock.contentcards.models import get_page_content_cards
+
+    def view_with_cards(request):
+        locale = l10n_utils.get_locale(request)
+        ctx = {'page_content_cards': get_page_content_cards('home', locale)}
+        return l10n_utils.render(request, 'sweet-words.html', ctx)
+
+The ``get_page_content_cards`` returns a dict of card data dicts for the given page (``home`` in this case)
+and locale. The dict keys are the names of the cards (e.g. ``card_1``). If the ``page_content_cards`` context
+variable is available in the template, then the ``content_card()`` macro will discover it automatically.
+
+.. note::
+
+    The ``get_page_content_cards`` function is not all that clever as far as l10n is concerned. If you have
```

Comment:
> you should have cards for every locale for which the page is active or the function will return an empty dict.

I'm not sure if having cards translated for every locale a page is active for is something we can realistically try to adhere to? Locales can be activated at any time, independent of changes to either bedrock or the www-admin repo. The warning you have here is great, but should we also be strongly encouraging developers to be explicit when passing `locale` into `get_page_content_cards`?

Reply:
It's a tough question/problem. Are we prepared to say that a template using these should only be locale specific, and then hardcode? That's fine, but a bit limiting. Perhaps we should recommend wrapping the cards section of the template in a language specification at least? 

```jinja
<% if LANG in ['de', 'fr', pt-BR'] %>
{# do cards #}
<% endif %>
```

Or decide in the view and only do cards if the `page_content_cards` variable is truthy?

Reply:
I updated the docs with an attempt. LMK what you think.

Reply:
Works for me 👍